ci/docker/conda-python-spark.dockerfile

```diff
@@ -20,21 +20,15 @@ ARG arch=amd64
 ARG python=3.6
 FROM ${repo}:${arch}-conda-python-${python}
 
-ARG jdk=8
-ARG maven=3.5
+ARG spark=3.0.1
 
 # The Spark tests currently break with pandas >= 1.0
```

Comment:
This shouldn't be the case any more

Reply:
Ok! I reverted the changes here and bumped pandas to the latest release (1.2) and hopefully the build will go green